Source/FuseJS/Bundle.uno

```diff
@@ -53,14 +53,15 @@ namespace FuseJS
 			}
 			```
 		*/
-		static Future<byte[]> GetBuffer(object[] args)
+		public static Future<byte[]> ReadBuffer(object[] args)
```

Comment:
I don't see a strong rationale for this. InternalsVisibleTo is targeted, this opens for more widespread use and thus dogfooding. I'm not convinced making these anything but public does any good.